<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sign_up</class>
 <widget class="QMainWindow" name="Sign_up">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sign_up</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(207, 207, 207);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(226, 226, 226);
border : 2px solid white;

font: 75 6pt &quot;맑은 고딕&quot;;</string>
      </property>
      <property name="title">
       <string>MEMBER LOGIN</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="3" rowspan="2">
        <widget class="QLabel" name="label_img">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(239, 239, 239);
border : 0px;</string>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="btn_login">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">border : 4px solid;
border-color : blue;
border-radius : 20px;
padding : 5px;

background-color : blue;
color: rgb(255, 255, 255);

font: 75 10pt &quot;맑은 고딕&quot;;</string>
         </property>
         <property name="text">
          <string>로그인</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="btn_join">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">border : 4px solid;
border-color : blue;
border-radius : 20px;
padding : 5px;

background-color : blue;
color: rgb(255, 255, 255);

font: 75 10pt &quot;맑은 고딕&quot;;</string>
         </property>
         <property name="text">
          <string>회원가입</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2" colspan="2">
        <widget class="QComboBox" name="comboBox">
         <property name="styleSheet">
          <string notr="true">border : 2px solid blue;
font: 75 10pt &quot;맑은 고딕&quot;;</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="3">
        <widget class="QLineEdit" name="edit_num">
         <property name="toolTip">
          <string/>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
border : 1px solid gray;


font: 75 10pt &quot;맑은 고딕&quot;;</string>
         </property>
         <property name="inputMask">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>학번</string>
         </property>
         <property name="maxLength">
          <number>32767</number>
         </property>
         <property name="frame">
          <bool>true</bool>
         </property>
         <property name="dragEnabled">
          <bool>false</bool>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
         <property name="placeholderText">
          <string/>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QLineEdit" name="edit_private_num">
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 75 10pt &quot;맑은 고딕&quot;;background-color: rgb(255, 255, 255);
border : 1px solid gray;</string>
         </property>
         <property name="text">
          <string>비밀번호</string>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>340</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
